Documentation/CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt

```diff
@@ -0,0 +1,39 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: json
+   :caption: EXT:site_package/composer.json
+   :linenos:
+
+   {
+   	"name": "typo3/site-package",
+   	"type": "typo3-cms-extension",
+   	"description": "Example site package from the site package tutorial",
+   	"authors": [
+   		{
+   			"name": "TYPO3 CMS Documentation Team",
+   			"role": "Developer",
+   			"homepage": "https://typo3.org/community/teams/documentation"
+   		},
+   		{
+   			"name": "The TYPO3 Community",
+   			"role": "Contributor",
+   			"homepage": "https://typo3.org/community/"
+   		}
+   	],
+   	"homepage": "https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage-Code",
+   	"license": "MIT",
+   	"keywords": [
+   		"typo3",
+   		"site package",
+   		"documentation"
+   	],
+   	"support": {
+   		"issues": "https://github.com/TYPO3-Documentation/t3docs-screenshots/issues"
+   	},
+   	"extra": {
+   		"typo3/cms": {
+   			"extension-key": "site_package"
+   		}
+   	}
+   }
+   
```

Comment:
The minimal requirement of the core is missing here see https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/ComposerJson/Index.html#minimal-composer-json

Reply:
We would have to change that in the extension. however. I will make an issue there

Reply:
https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage-Code/issues/5